en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+- Rendering the IDV Provider templates page in the console.
+- Rendering the thumbnail image for each IDV provider in the console.
+
+**Structure**
+
+The following fields should be available on the info.json.
+
+| Field        | Description                                                                                                                     | 
+|--------------|---------------------------------------------------------------------------------------------------------------------------------| 
+| id           | Unique id for the identity verification provider type.                                                                          |
+| name         | Name of the identity verification provider type.                                                                                |
+| description  | A short description of the identity verification provider type.                                                                 |
+| image        | An image URL for the identity verification provider.                                                                            |
+| displayOrder | Not utilized in the IDV provider UI. But an integer needs to be included for compatibility with extensions APIs.                |
+| tags         | Tags associated with the identity verification provider. Can be used to filter providers based on the tags from the console UI. |
+| category     | Not utilized in the IDV provider UI. But a string needs to be included for compatibility with extensions APIs.                  |
+| type         | Extension resource type. “identity-verification-provider” should be used as the value.                                          |
+
+**Example**
+
+```json
+{
+  "id": "ONFIDO",
+  "name": "Onfido",
+  "description": "Enable identity verification through Onfido",
+  "image": "https://example.com/icon.png",
+  "tags": ["Enterprise"],
+  "category": "DEFAULT",
+  "displayOrder": 1,
+  "type": "identity-verification-provider"
+}
+```
+
+2. ### Create a metadata.json file
+
+The metadata.json file contains the UI metadata that is required for rendering the dynamically generated content. Currently, this file is used when rendering the
+- Configuration section in the IDV Provider creation wizard in the console.
+- Settings section in the IDV Provider edit page in the console.
+- Configuration Properties section in the IDV Provider create/edit pages in the management console.
+
+The UI metadata defines what content needs to be rendered, how it should be structured, and what validations need to be performed.
+
+**Structure**
+
+The metadata.json file should contain the following structure.
+```
+{
+    "common": {
+    "configProperties": [ <An array of input fields> ]
+    }
+}
+```
+**Supported Input Types**
+
+Currently the following input field types are supported.
+
+| Input Field Type | Description                                                                          | Supported In |
```

Comment:
All input types except "toggle" are supported in the management console as well. Toggle is rendered as a checkbox.

Reply:
Resolved